diff --git a/src/core/md_os.h b/src/core/md_os.h
index 3b60174..39d715c 100644
--- a/src/core/md_os.h
+++ b/src/core/md_os.h
@@ -16,6 +16,10 @@ extern "C" {
 // Returns a non owning reference to a shared static string buffer
 str_t   md_path_cwd(void);
 
+// Gets the directory containing the running executable (with trailing slash).
+// Returns a non-owning reference to a shared static string buffer.
+str_t   md_path_exe_dir(void);
+
 str_t   md_path_make_canonical(str_t path, struct md_allocator_i* alloc);
 str_t   md_path_make_relative(str_t path_from, str_t path_to, struct md_allocator_i* alloc);
 
