--- openspace.cfg	2025-08-21 20:41:44.793491957 +0530
+++ openspace6b0de70.cfg.edited	2025-08-21 20:41:48.682572705 +0530
@@ -67,21 +67,21 @@
   FONTS = "${DATA}/fonts",
   TASKS = "${DATA}/tasks",
   -- If the the 'OPENSPACE_SYNC' environment variable is defined on the system, use that
-  -- value.  Otherwise, fall back to the ${BASE}/sync folder instead.  This allows a
+  -- value.  Otherwise, fall back to the ${USER}/sync folder instead.  This allows a
   -- reuse of the sync folder between multiple OpenSpace installations by simply setting
   -- that environment variable
-  SYNC = os.getenv("OPENSPACE_SYNC") or "${BASE}/sync",
+  SYNC = os.getenv("OPENSPACE_SYNC") or "${USER}/sync",
   SCREENSHOTS = "${USER}/screenshots",
   WEB = "${DATA}/web",
   RECORDINGS = "${USER}/recordings",
-  CACHE = "${BASE}/cache",
+  CACHE = "${USER}/cache",
   CONFIG = "${BASE}/config",
   DOCUMENTATION = "${BASE}/documentation",
-  LOGS = "${BASE}/logs",
+  LOGS = "${USER}/logs",
   MODULES = "${BASE}/modules",
   SCRIPTS = "${BASE}/scripts",
   SHADERS = "${BASE}/shaders",
-  TEMPORARY = "${BASE}/temp",
+  TEMPORARY = "${USER}/temp",
   GLOBEBROWSING = os.getenv("OPENSPACE_GLOBEBROWSING") or "${USER}/globebrowsing"
 }
 
@@ -89,7 +89,7 @@
   GlobeBrowsing = {
     TileCacheSize = 2048, -- for all globes (CPU and GPU memory)
     MRFCacheEnabled = false,
-    MRFCacheLocation = (os.getenv("OPENSPACE_GLOBEBROWSING") or "${BASE}") .. "/mrf_cache",
+    MRFCacheLocation = (os.getenv("OPENSPACE_GLOBEBROWSING") or "${USER}") .. "/mrf_cache",
     DefaultGeoPointTexture = "${DATA}/globe_pin.png"
   },
   Sync = {
